lib/complexValues/function.js

```diff
@@ -69,34 +63,6 @@ class DescribedFunctionValue extends object.DescribedMixin(FunctionValue) {
     return super.compare(expected)
   }
 
-  createPropertyRecursor () {
-    const recursor = super.createPropertyRecursor()
-
-    const skipName = this.nameIsEnumerable
-    if (!skipName) return recursor
-
-    let size = recursor.size
-    if (skipName) {
-      size -= 1
-    }
-
-    if (size === 0) return NOOP_RECURSOR
-
-    const next = () => {
-      const property = recursor.next()
-      if (property) {
-        if (skipName && property.key.value === 'name') {
-          return next()
-        }
-        return property
-      }
-
-      return null
-    }
-
-    return { size, next }
-  }
```

Comment:
I'd prefer to keep this code for now though, even if we can't test it. It dates back to when there was a supported Node.js version which behaved differently with function names. But there's probably other compatibility code around this, so let's keep this until we can remove it properly everywhere.

Reply:
I restored the existing behavior.